src/main/resources/templates/send.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layout.html}">
+<head>
+    <meta charset="UTF-8">
+    <title>Formularz Kontaktowy</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
+</head>
+<body>
+<section layout:fragment="content">
+    <form action="/contact" method="post">
+        <div style="width: 500px; margin: 50px auto; margin-top: 100px; text-align: center">
```

Comment:
Staraj się unikać takich inlinowych styli, szczególnie jeżeli korzystasz z bootstrapa, to nie powinno to być raczej potrzebne